AddCustomer_Coding.txt

```diff
@@ -0,0 +1,150 @@
+public class AddCustomer extends javax.swing.JDialog {
+
+    public AddCustomer() {
+        initComponents();
+        setLocationRelativeTo(this);
+    }
+
+    @SuppressWarnings("unchecked")
+    // <editor-fold defaultstate="collapsed" desc="Generated Code">                          

```

Comment:
This code is the function or event let the staff to add the new customer into the system.